spec/features/users_can_report_a_repair_without_previous_answers_affecting_result_spec.rb

```diff
@@ -0,0 +1,166 @@
+      'propertyReference' => '00000512',
+      'address' => 'Ross Court 42',
+      'postcode' => 'E5 8TE',
+    }
+    fake_api = instance_double(JsonApi)
+    allow(fake_api).to receive(:get)
+      .with('v1/properties?postcode=E5 8TE')
+      .and_return('results' => [property])
+    allow(fake_api).to receive(:get)
+      .with('v1/properties/00000512')
+      .and_return(property)
+    allow(fake_api).to receive(:post)
+      .with('v1/repairs', anything)
+      .and_return(
+        'repairRequestReference' => '00367923',
+        'priority' => 'N',
+        'problemDescription' => 'My sink is blocked',
+        'propertyReference' => '00000503',
+        'workOrders' => [
+          {
+            'sorCode' => '0078965',
+            'workOrderReference' => '09124578',
+          },
+        ],
+      )
+    allow(fake_api).to receive(:get)
+      .with('v1/repairs/00367923')
+      .and_return(
+        'repairRequestReference' => '00367923',
+        'orderReference' => '09124578',
+        'priority' => 'N',
+        'problemDescription' => 'My sink is blocked',
+        'propertyReference' => '00000503',
+        'workOrders' => [
+          {
+            'sorCode' => '0078965',
+            'workOrderReference' => '09124578',
+          },
+        ],
+      )
+    allow(fake_api).to receive(:get)
+      .with('v1/work_orders/09124578/available_appointments')
+      .and_return(
+        'results' => [
+          {
+            'beginDate' => '2017-11-27T10:00:00Z',
+            'endDate' => '2017-11-27T12:00:00Z',
+            'bestSlot' => true,
+          },
+        ]
+      )
+    allow(fake_api).to receive(:post)
+      .with('v1/work_orders/09124578/appointments', anything)
+      .and_return(
+        'beginDate' => '2017-11-27T10:00:00Z',
+        'endDate' => '2017-11-27T12:00:00Z'
+      )
+    allow(JsonApi).to receive(:new).and_return(fake_api)
+
+    fake_question_set = instance_double(QuestionSet)
+    allow(fake_question_set)
+      .to receive(:find)
+      .with('location')
+      .and_return(
+        Question.new(
+          'question' => 'Where is the problem?',
+          'answers' => [
+            { 'text' => 'Kitchen', 'next' => 'kitchen' },
+            { 'text' => 'Bathroom' },
+            { 'text' => 'Other' },
+          ],
+        )
+      )
+    allow(fake_question_set)
+      .to receive(:find)
+      .with('kitchen')
+      .and_return(
+        Question.new(
+          'question' => 'Is your tap broken?',
+          'answers' => [
+            { 'text' => 'Yes', 'sor_code' => '0078965' },
+            { 'text' => 'No' },
+          ],
+        )
+      )
+    allow(QuestionSet).to receive(:new).and_return(fake_question_set)
+
+    visit '/'
+    click_on 'Start'
+
+    # Emergency page:
+    choose_radio_button 'No'
+    click_on 'Continue'
+
+    # Fake decision tree
+    choose_radio_button 'Kitchen'
+    click_on 'Continue'
+
+    choose_radio_button 'No' # Doesn't get a SOR code
```

Comment:
Currently this spec verifies the behaviour by checking what is shown on the confirmation page. Would it be clearer and shorter to check that when the user gets an SOR code _first_ then they see the appointments page when requesting a second repair that doesn't have an SOR code?